settings.gradle

```diff
@@ -76,6 +76,15 @@ List projects = [
   'qa:wildfly'
 ]
 
+File examplePluginsDir = new File(rootProject.projectDir, 'plugins/examples')
```

Comment:
I did this for two reasons:
1. Otherwise all the places that list plugins (eg smoke-test-plugins) via the gradle project `:plugins` must add a bunch of logic we must copy every time we want to get all the plugins.
2. I have come around to hating having to list every plugin/module, as it is both cumbersome to change, as well as error prone because moving/creating a new module results in a cryptic error message from gradle until it is added (thinking that the project is standalone instead of our multi project build).